packages/generators/generators/component.js

```diff
@@ -0,0 +1,23 @@
+import { actions, prompts } from './constants.js';
+
+const componentGenerator = {
+	description: 'Add a component package',
```

Comment:
What is a component package? I thought components in the monorepo are supposed to live in `@automattic/components`?

Reply:
I've updated copy to `Add a new library of React components designed for use in Automattic products`, WDYT? We can iterate this in the follow-up PR

Reply:
Sounds better, although I'm still not convinced there's a use case for a similar instance of a package, given that we already have `@automattic/components`.

Reply:
I think there are examples of larger features (which are react components) which exist outside of @automattic/components. For example, `@automattic/page-pattern-modal`

Reply:
I see, thanks for clarifying. I guess it's a matter of perspective, since everything in React is either a hook or a component. Let's leave this as-is 👍